engine/class_modules/monk/sc_monk.cpp

```diff
@@ -462,14 +462,14 @@ void monk_action_t<Base>::consume_resource()
     {
       if ( p()->talent.shado_pan.flurry_strikes.ok() )
       {
-        // Tiger Palm counts the full energy base cost
-        if ( base_t::id == p()->baseline.monk.tiger_palm->id() )
-          p()->flurry_strikes_energy += as<int>( base_t::base_cost() );
-        // Crackling Jade Lightning is whatever the current cost
-        else if ( base_t::id == p()->baseline.monk.crackling_jade_lightning->id() )
+        // Tiger Palm, Keg Smash, and Crackling Jade Lightning contribute their current cost to the accumulator
+        if ( base_t::id == p()->baseline.monk.tiger_palm->id() ||
+             base_t::id == p()->talent.brewmaster.keg_smash->id() ||
+             base_t::id == p()->baseline.monk.crackling_jade_lightning->id() )
           // this needs to be rounded to the nearest whole number
           p()->flurry_strikes_energy += as<int>( std::round( final_cost ) );
```

Comment:
You can use `std::lround( final_cost );` to directly produce the desired value, and it should not warn.
<https://en.cppreference.com/w/cpp/numeric/math/round>